contacts/decorators.py

```diff
@@ -0,0 +1,19 @@
+from functools import wraps
+from django.core.exceptions import PermissionDenied
+
+from .models import Contact, Address
+
+
+def user_added_contact(view_func):
```

Comment:
this decorator doesn't work just for `contacts` but also for `addresses`. please think of some generic name keeping its functionality in mind!